src/main/java/snowy/data/SnowyException.java

```diff
@@ -0,0 +1,7 @@
+package snowy.data;
```

Comment:
Would be better if the package is named exceptions for easier understandability of code organization